app/src/main/AndroidManifest.xml

```diff
@@ -139,7 +139,15 @@
             android:name="org.thoughtcrime.securesms.dms.CreatePrivateChatActivity"
             android:screenOrientation="portrait"
             android:windowSoftInputMode="adjustResize"
-            android:theme="@style/Theme.Session.DayNight.FlatActionBar" />
+                  android:theme="@style/Theme.Session.DayNight.FlatActionBar">
+        
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="loki"/>
```

Comment:
Seems like this should probably be `session` rather than `loki`

Reply:
`session` already in use, that's why I used `loki`

Reply:
I think we will have to synchronise this across the iOS and Desktop platforms, but `loki` might not be an ideal fit. We may be able to use a combination of either `session://` with a specific host if that avoids conflicts, or even a separate scheme that works across the considerations for what's allowed on Desktop for example. Thanks for the PR @privatgt